fetch.bs

```diff
@@ -4855,6 +4855,70 @@ initially a new {{AbortSignal}} object.
 <a for=Request>request</a>'s
 <a for=request>body</a>.
 
+<dl class=domintro>
+ <dt><code><var>request</var> = new <a constructor lt="Request()">Request</a>(<var>input</var> [,
+ <var>init</var>])</code>
+ <dd>Returns a new <var>request</var> whose <a attribute for=Request>url</a> attribute value is
+ <var>input</var> if <var>input</var> is a string, and <var>input</var>'s
+ <a attribute for=Request>url</a> if <var>input</var> is a {{Request}}. The optional
+ <var>init</var> argument allows for setting attributes appearing in {{RequestInit}} via object
+ members of the same name.
+
+ <dt><code><var>request</var> . <a attribute for=Request>method</a></code>
+ <dd>Returns <var>request</var>'s HTTP method, which is "<code>GET</code>" by default.
+
+ <dt><code><var>request</var> . <a attribute for=Request>url</a></code>
+ <dd>Returns the URL of <var>request</var> as a string.
+
+ <dt><code><var>request</var> . <a attribute for=Request>headers</a></code>
+ <dd>Returns a {{Headers}} object consisting of the headers associated with <var>request</var>.
+
+ <dt><code><var>request</var> . <a attribute for=Request>destination</a></code>
+ <dd>Returns the kind of resource requested by <var>request</var>, e.g. "<code>document</code>",
+ or "<code>script</code>".
+
+ <dt><code><var>request</var> . <a attribute for=Request>referrer</a></code>
+ <dd>Returns the referrer of <var>request</var>. Its value can be a same origin URL if explicitly
+ set in <var>init</var>, the empty string to indicate no referrer, and "<code>about:client</code>"
+ when defaulting to the document's <a for=Document>URL</a>. This is used during fetching to
```

Comment:
This isn't really accurate for when you'd use fetch() in workers. I wonder how specific we want to get. I think I'd be okay with "the global's default" or some such.

Reply:
Ok that works. If in a worker, is the condition in [determine request's referrer](https://w3c.github.io/webappsec-referrer-policy/#determine-requests-referrer) "client" > step 1 not met? (thus defaulting to the creation URL?)

Reply:
Yeah. A worker has a `WorkerGlobalScope` object as global object.